src/modules/itemTypes.js

```diff
@@ -57,16 +57,27 @@ export const getPlugin = type => {
 
 // Item type map
 export const itemTypeMap = {
+    [VISUALIZATION]: {
+        id: VISUALIZATION,
+        endPointName: 'visualizations',
+        propName: 'visualization',
+        pluralTitle: i18n.t('Visualizations'),
+        plugin: 'visualizationPlugin',
+        domainType: DOMAIN_TYPE_AGGREGATE,
+        visualizationType: VISUALIZATION_TYPE_TABLE,
```

Comment:
I believe this is used in the item header button for switching between chart/pt/map.
Do we still need that functionality for charts/PT? Now we would need to change visType within the vis object passed to the plugin.
Also, the icon here cannot be "fixed" it depends on the current visType of the dashboard item.